mne/report/tests/test_report.py

```diff
@@ -639,6 +639,42 @@ def test_add_or_replace(tags):
     assert r.html[3] == old_r.html[3]
 
 
+def test_add_or_replace_section():
+    """Test that sections are respected when adding or replacing."""
+    r = Report()
+    fig1, fig2 = _get_example_figures()
+    r.add_figure(fig=fig1, title='a', section='A')
+    r.add_figure(fig=fig1, title='a', section='B')
+    r.add_figure(fig=fig1, title='a', section='C')
+    # By default, replace=False, so all figures should be there
+    assert len(r.html) == 3
+    assert len(r._content) == 3
+
+    old_r = copy.deepcopy(r)
+    assert r.html[0] == old_r.html[0]
+    assert r.html[1] == old_r.html[1]
+    assert r.html[2] == old_r.html[2]
+
+    # Replace last occurrence of `fig1` tagges as `foo`
```

Comment:
I think I understand what the test is doing but I don't understand this comment; seems like you're replacing the *middle of 3* occurrences of a figure (not the *last* occurrence), and I don't see where "foo" comes in to it.  Also `tagges` -> `tagged`?

Reply:
Probably an old comment, will update